src/renderers/common/XRManager.js

```diff
@@ -1036,6 +1036,7 @@ class XRManager extends EventDispatcher {
 					}
 				);
 
+				this._xrRenderTarget._isOpaqueFramebuffer = true;
```

Comment:
Do you mind declaring this property in `XRRenderTarget` as a new public property? 

Since it is only relevant for XR, I would prefer to move it there. The WebGL backend should then check if `isXRRenderTarget` is `true` and only then check `isOpaqueFramebuffer`.